inst/sql/sql_server/CreateResultsDataModel.sql

```diff
@@ -362,9 +362,10 @@ CREATE TABLE @results_schema.@index_event_breakdown (
 
 CREATE TABLE @results_schema.@metadata (
 			database_id VARCHAR NOT NULL,
+			start_time VARCHAR NOT NULL,
 			variable_field VARCHAR NOT NULL,
 			value_field VARCHAR,
-			PRIMARY KEY(database_id, variable_field)
+			PRIMARY KEY(database_id, start_time, variable_field)
```

Comment:
will this break backwards compatibility with old runs where metadata.csv doesn't contain a start time?
Could it be made not required and default to something like 0:0:0?

Reply:
I think it can be defaulted to 0:0:0

Reply:
Do you know to modify the ddl so that it always has a default value?

Reply:
Will it break DiagnosticExplorer - metadata is not used by DiagnosticExplorer right now. its mostly to explore the content as csv

Reply:
```
start_time VARCHAR DEFAULT '0:0:0',
```

Should work